ckanext/dia_theme/templates/snippets/package_item.html

```diff
@@ -8,7 +8,7 @@
 {% block resources_inner %}
   {% for resource in h.dict_list_reduce(package.resources, 'format') %}
     <li>
-      <a href="{{ h.url_for(controller='package', action='read', id=package.name) }}#dataset-resources" class="label" data-format="{{ resource.lower() }}">{{ resource }}</a>
+      <a href="{{ h.url_for(controller='resource' if h.ckan_version().split('.')[1] | int >= 9 else 'package', action='read', id=package.name) }}#dataset-resources" class="label" data-format="{{ resource.lower() }}">{{ resource }}</a>
```

Comment:
I wish they'd provided a helper method for this, feels super gross spraying this check everywhere.

Reply:
I suppose any helper would have to exist in the older versions too though.